base/compiler/abstractinterpretation.jl

```diff
@@ -1199,7 +1213,20 @@ function abstract_call_known(interp::AbstractInterpreter, @nospecialize(f),
 end
 
 function abstract_call_opaque_closure(interp::AbstractInterpreter, closure::PartialOpaque, argtypes::Vector{Any}, sv::InferenceState)
-    return CallMeta(Any, nothing)
+    pushfirst!(argtypes, closure.env)
+    sig = argtypes_to_type(argtypes)
+    rt, edgecycle, edge = abstract_call_method(interp, closure.source::Method, sig, Core.svec(), false, sv)
+    info = OpaqueClosureCallInfo(edge)
```

Comment:
JET now warns me `edge` could be `nothing` and thus this call may result in an error.
I guess we may want to bail out `return CallMeta(rt, false)` in that case ?

Reply:
Yeah, I'm revising this code in a follow up PR